### PR TITLE
Give the project a name

### LINKED ISSUE #1: Name it already!
We need a project name!

### PR DESCRIPTION
**Pull request by omriAckley**
Closes #1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "__BONES_NEEDS_A_NAME__",
+  "name": "boneyard",
```
Comment (johnptmcdonald) [inline package.json]: @omriBernstein I thought we agreed on 'graveyard'? can you check?

**Review by johnptmcdonald — CHANGES_REQUESTED** [review]
please see all my individual comments

File: package.json (hunk 1 above)
Comment (omriAckley) [reply package.json]: Oops, changing it

**Comment by omriAckley** [comment]
@johnptmcdonald looks good?

**Review by johnptmcdonald — APPROVED** [review]

**Pull request merged**